blocks/blogheader/blogheader.css

```diff
@@ -0,0 +1,150 @@
+main.has-sidebar > div.blogheader-container {
+  grid-column: 2 / span 2;
+  color: var(--background-color);
+  padding: 0;
+}
+
+nav#blogheader {
+  --number-of-menu-items: 5;
+
+  padding: 0;
+  margin: 0;
+}
+
+nav#blogheader > div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav#blogheader ul {
+  display: table;
+  table-layout: fixed;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  background-color: var(--text-color);
+}
+
+nav#blogheader li a {
+  font-size: 1.4rem;
+  text-decoration: none;
+  transition: none;
+  background: inherit;
+  color: inherit;
+  font-weight: 400;
+}
+
+nav#blogheader li {
+  display: table-cell;
+  border-right: 1px solid white;
+  color: white;
+  text-align: center;
+  padding: 1.5rem;
+  width: calc(55rem / var(--number-of-menu-items));
+}
+
+nav#blogheader li.blogsearch-menu-container {
+  border: none;
+  text-align: right;
+  width: auto;
+  display: flex;
+  justify-content: flex-end;
+}
+
+nav#blogheader li.active {
+  background-color: black;
+}
+
+/* Style for the paragraph */
+nav#blogheader p {
+  margin: 0;
+  padding: 0;
+}
+
+@media (min-width: 900px) {
```

Comment:
their switch point is `768px`, I think we should use the same.

Reply:
thanks for noting that, changed this to 768